src/app/portlets/content-types/main/content-types.component.scss

```diff
@@ -0,0 +1,6 @@
+
+tbody {
```

Comment:
The ```!important``` it's not necessary.

Also, this is affecting the whole app so:

1. Remove the ```ViewEncapsulation.None``` in the ```content-types.component```
2. Use the encapsulation to style this.